test/other/codesize/test_codesize_hello_single_file.jssize

```diff
@@ -1 +1 @@
-6690
```

Comment:
Do you know why code size changes are part of this PR ?   It looks like it should only effect `-sSAFE_HEAP` programs?

Reply:
I was surprised by it too but I long gave up trying to understand why code sizes are behaving the way they are by now 😅

Reply:
Looks like CI disagrees with my rebaseline, so I guess I need to update tot yet again.

Is this something we can simplify? Why do we rely on developer having latest tot rather than just use local Emscripten checkout we are in?

Reply:
Ah I guess it's because local checkout doesn't have binary deps? Can test runner download those separately into local checkout maybe?

Reply:
Isn't that basically that `emsdk install tot` does?   

Its pretty simple to use those binaryies, from your emscripten checkout, my preferred way is to just do `export EM_CONFIG=/path/to/emsdk/.emscripten` and then run `./test/runner` and it will use the emsdk binaries.

One followup I was intending to do is have the CI action be able to post the updated results back to the PR so you can just "accept" the results of the CI machine if you prefer.  Not sure how hard that would be though.

Reply:
> Isn't that basically that emsdk install tot does?

Yeah I just meant to have it 1) be done automatically and 2) isolated from my "normal" Emscripten setup as I kinda want stable Emscripten release when not contributing but just using it.

> One followup I was intending to do is have the CI action be able to post the updated results back to the PR so you can just "accept" the results of the CI machine if you prefer. Not sure how hard that would be though.

That would be amazing tbh.